charts/GPP-stack/values.yaml

```diff
@@ -0,0 +1,831 @@
+      # axes: openzaak-redis-master-0:6379/0
+      # portalLocker: openzaak-redis-master-0:6379/0
+      default: ""
+      axes: ""
+      portalLocker: ""
+
+    celery:
+      # -- Deploy celery (worker), celery is required from openzaak version 1.8.0
+      enabled: true
+      # Celery cache settings, when the redis subchart is enabled these settings will be ignored and caching will default to the redis subchart services
+      # e.g.:
+      # brokerUrl: redis://openzaak-redis-master:6379/1
+      # resultBackend: redis://openzaak-redis-master:6379/1
+      brokerUrl: ""
+      resultBackend: ""
+      logLevel: debug
+
+    notificationsDisabled: false
+
+    jwtExpiry: 3600
+
+    cmis:
+      enabled: false
+      mapperFile: ""
+
+    isHttps: true
+
+    debug: false
+
+    uwsgi:
+      master: ""
+      threads: ""
+      processes: ""
+      maxRequests: ""
+      harakiri: ""
+
+  worker:
+    replicaCount: 1
+    concurrency: 4
+    podLabels: {}
+    resources: {}
+    livenessProbe:
+      initialDelaySeconds: 60
+      periodSeconds: 10
+      timeoutSeconds: 5
+      failureThreshold: 3
+      successThreshold: 1
+    readinessProbe:
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      timeoutSeconds: 5
+      failureThreshold: 3
+      successThreshold: 1
+    autoscaling:
+      enabled: false
+      minReplicas: 1
+      maxReplicas: 100
+      targetCPUUtilizationPercentage: 80
+      targetMemoryUtilizationPercentage: 80
+
+  nginx:
+    image:
+      repository: nginxinc/nginx-unprivileged
+      pullPolicy: IfNotPresent
+      tag: stable
+    service:
+      type: ClusterIP
+      port: 80
+      annotations: {}
+    # -- mount existing nginx vhost config
+    existingConfigmap: null
+    replicaCount: 1
+    podLabels: {}
+    securityContext:
+      capabilities:
+        drop:
+        - ALL
+      readOnlyRootFilesystem: false
+      runAsNonRoot: true
+      runAsUser: 101
+    autoscaling:
+      enabled: false
+    livenessProbe:
+      initialDelaySeconds: 60
+      periodSeconds: 10
+      timeoutSeconds: 5
+      failureThreshold: 3
+      successThreshold: 1
+    readinessProbe:
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      timeoutSeconds: 5
+      failureThreshold: 3
+      successThreshold: 1
+    resources: {}
+
+  flower:
+    enabled: true
+    replicaCount: 1
+    podLabels: {}
+    livenessProbe:
+      initialDelaySeconds: 60
+      periodSeconds: 10
+      timeoutSeconds: 5
+      failureThreshold: 6
+      successThreshold: 1
+    readinessProbe:
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      timeoutSeconds: 5
+      failureThreshold: 6
+      successThreshold: 1
+    resources: {}
+
+  # Redis subchart
+  redis:
+    architecture: standalone
+    auth:
+      enabled: false
+    master:
+      persistence:
+        enabled: false
+      resources:
+        requests:
+          cpu: 10m
+          memory: 20Mi
```

Comment:
Todo:
- I need to figure out why this doesn't work unless I add 
```
nameOverride: "openzaak-redis"
sentinel:
    masterService:
    enabled: false
```
(and in the gpp-publicatiebank just the name override)